internal/participation.go

```diff
@@ -127,3 +94,42 @@ func RemovePartKeyByID(slice *[]api.ParticipationKey, id string) {
 		}
 	}
 }
+
+func FindParticipationIdForVoteKey(slice *[]api.ParticipationKey, votekey []byte) *string {
+	for _, item := range *slice {
+		if string(item.Key.VoteParticipationKey) == string(votekey) {
+			return &item.Id
+		}
+	}
+	return nil
+}
+
+func ToLoraDeepLink(network string, offline bool, part api.ParticipationKey) (string, error) {
+	fee := 2000000
+	var loraNetwork = strings.Replace(strings.Replace(network, "-v1.0", "", 1), "-v1", "", 1)
```

Comment:
Does this work for FNet? Its genesis ID is `fnet-v1`

Reply:
Yea should be handled, Lora uses "fnet/*" as the route parameter and we just strip the -v1

Reply:
Not sure how I missed the nested replacing when I saw this.